entity/contents/snapshot.h

```diff
@@ -0,0 +1,33 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#pragma once
+
+#include <functional>
+#include <memory>
+#include <vector>
+
+#include "flutter/fml/macros.h"
+#include "impeller/geometry/matrix.h"
+#include "impeller/geometry/rect.h"
+#include "impeller/renderer/texture.h"
+
+namespace impeller {
+
+class ContentContext;
+class Entity;
+
+/// Represents a screen space texture and its intended draw position.
```

Comment:
Why should this struct care if this position is in screen space?

Reply:
Removed the "screen space" mention.

Right now it always ends up being flat screen space in practice, so I wanted to curb potential confusion that might come up about what the position/size means exactly in the context of where it's used in code. It's true that this won't always necessarily be the case in the future, though.